app/models/game.rb

```diff
@@ -4,4 +4,32 @@ class Game < ApplicationRecord
   has_one :black_player, class_name: 'User', dependent: false
   has_one :winning_player, class_name: 'User', dependent: false
   has_many :pieces, dependent: false
+
+  def populate_board!
+    Rook.create(position_x: 0, position_y: 0, game: self, color: 'black')
+    Knight.create(position_x: 0, position_y: 1, game: self, color: 'black')
+    Bishop.create(position_x: 0, position_y: 2, game: self, color: 'black')
+    Queen.create(position_x: 0, position_y: 3, game: self, color: 'black')
+    King.create(position_x: 0, position_y: 4, game: self, color: 'black')
+    Bishop.create(position_x: 0, position_y: 5, game: self, color: 'black')
+    Knight.create(position_x: 0, position_y: 6, game: self, color: 'black')
+    Rook.create(position_x: 0, position_y: 7, game: self, color: 'black')
+
+    Rook.create(position_x: 7, position_y: 0, game: self, color: 'white')
+    Knight.create(position_x: 7, position_y: 1, game: self, color: 'white')
+    Bishop.create(position_x: 7, position_y: 2, game: self, color: 'white')
+    Queen.create(position_x: 7, position_y: 3, game: self, color: 'white')
+    King.create(position_x: 7, position_y: 4, game: self, color: 'white')
+    Bishop.create(position_x: 7, position_y: 5, game: self, color: 'white')
+    Knight.create(position_x: 7, position_y: 6, game: self, color: 'white')
+    Rook.create(position_x: 7, position_y: 7, game: self, color: 'white')
+
+    0.upto(7) do |b| # b to stand in for black for now
+      Pawn.create(position_x: 1, position_y: b, game: self, color: 'black')
+    end
+
+    0.upto(7) do |w| # w to stand in for white for now
+      Pawn.create(position_x: 6, position_y: w, game: self, color: 'white')
+    end
+  end
```

Comment:
Looking good!

- did you add a color column to piece?
- Try adding a few tests for this

Reply:
Thanks! I did add a color column, but I did that on `branch render-pieces`. I'll open an PR on that one now, for you to review. I'm still trying to figure out that helper method, but if you wouldn't mind taking a look at the schema.rb file in that branch, somehow I deleted a lot of columns, but don't believe I touched that. I'll make a comment over there so you can see what I mean. Once I render a piece I'll add some tests for this - I'm determined to render a piece! haha